### PR TITLE
fix APC(u) cache storage

### PR DESCRIPTION
**Pull request by Fedik**
this fix for APC caching when server use PHP 5.5 with APCu,
in my configuration it:

PHP 5.5.9
APCu 4.0.2

**how to test**
make sure that you have installed PHP 5.5 (or higher) and APCu extension,
enable APC caching in Global Configuration, and go to the "Clean Cache" page, there you will see a lot errors,

then apply patch, and again go to the "Clean Cache" page and check, whether you are now able to see the cache info, and able to clean up the cache

PS: it seems bug in PHP APCu extension,  in compatibility with old APC

### CODE REVIEW

File: libraries/joomla/cache/storage/apc.php [hunk 1]
```diff
@@ -135,9 +137,10 @@ public function clean($group, $mode = null)
 
 		foreach ($keys as $key)
 		{
-			if (strpos($key['info'], $secret . '-cache-' . $group . '-') === 0 xor $mode != 'group')
+			$name = !empty($key['info']) ? $key['info'] : $key['key'];
+			if (strpos($name, $secret . '-cache-' . $group . '-') === 0 xor $mode != 'group')
```
Comment (zero-24) [inline libraries/joomla/cache/storage/apc.php]: @Fedik can you add a empty line between the `if` and the `$name`?

File: libraries/joomla/cache/storage/apc.php [hunk 2]
```diff
@@ -159,9 +162,10 @@ public function gc()
 
 		foreach ($keys as $key)
 		{
-			if (strpos($key['info'], $secret . '-cache-'))
+			$name = !empty($key['info']) ? $key['info'] : $key['key'];
+			if (strpos($name, $secret . '-cache-'))
```
Comment (zero-24) [inline libraries/joomla/cache/storage/apc.php]: same here. Thanks!

**Comment by Fedik** [comment]
@zero-24 fixed :wink:

**Comment by Fedik** [comment]
fixed in #9330

**Pull request closed without merging**